great-idea/services.html

```diff
@@ -15,93 +15,121 @@
 </head>
 
 <body>
-
-
-Services
-Product
-Vision
-Features
-About
-Contact
-
-<img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
-
-
-<img class="services-header-img" src="img/services-header.jpg" alt="Our services header image">
-
-Services
+  <header>
+    <nav class="navigation">
+      <a href="services.html">Services</a>
+      <a href="">Product</a>
+      <a href="">Vision</a>
+      <a href="">Features</a>
+      <a href="">About</a>
+      <a href="">Contact</a>
+      <a href="index.html" >
+       <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
+      </a>
+      </nav>
+    </header>
+ <div class="container">
+  <img class="services-header-img" src="img/services-header.jpg" alt="Our services header image">
+
+<div class="serviceHead">
+<h1>Services</h1>
 
 Our services provide the best digital value in nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque eleifend. Mauris euismod facilisis iaculis.
```

Comment:
These blocks of text should be wrapped in something, possibly a `<p>` tag. You will almost never want plain text to be in your HTML without using a tag to wrap the content